docs/GETTING_STARTED_GUIDE.md

```diff
@@ -22,6 +22,14 @@ First and foremost, the metrics service requires a Kubernetes cluster that align
 | --------- |
 | 1.17-1.19 |
 
+## Dell EMC CSI Driver for PowerFlex
+
+This porject captures telemetry data about Kubernetes storage usage and performance which are provisioned through Dell EMC CSI (Container Storage Interface) Driver for PowerFlex. The metrics service requires CSI Driver for PowerFlex is deployed in Kubenetes cluster.
+
+| CSI Driver |
+| ---------- |
+| [CSI Driver for PowerFlex v1.1.5, 1.2.0, 1.2.1](https://github.com/dell/csi-vxflexos) |
```

Comment:
We have this information in the README too. Do we want to remove it from there and keep it here? Otherwise, we'll have to remember to update both places whenever we have add more supported revs.

Reply:
I added it here, as I felt telling user that they need to deploy CSI driver, without version is bit incomplete